### ISSUE TITLE
wallet-api: response returns wrong session id if it's bigger than INT_MAX(2147483647)

### ISSUE THREAD

**Issue by andreibaranouski** [BeamMW/beam#866]
**To Reproduce**
request to wallet-api works fine with INT_MAX C++: 2147483647
```
$ curl -d '{"jsonrpc":"2.0","id": 2147483647,"method":"wallet_status"}' -H "Content-Type: application/json" -X POST http://127.0.0.1:10006/api/wallet | python -m json.tool

{
    "id": 2147483647,
    "jsonrpc": "2.0",
    "result": {
        "available": 0,
        "current_height": 295384,
        "current_state_hash": "bd04f38fbedcae7ce088955dc01d2cab5bea1f5af118d7134aa4047b3a726803",
        "difficulty": 776.5170288085938,
        "maturing": 0,
        "prev_state_hash": "f2f099fd72959750ee4c4a96b588b0a657f16562e7c3acded1d946ec8957b4ac",
        "receiving": 0,
        "sending": 0
    }
}
```

 but doesn't work with bigger value 2147483648 and returns negative "id": -2147483648

```
$ curl -d '{"jsonrpc":"2.0","id": 2147483648,"method":"wallet_status"}' -H "Content-Type: application/json" -X POST http://127.0.0.1:10006/api/wallet | python -m json.tool

{
    "id": -2147483648,
    "jsonrpc": "2.0",
    "result": {
        "available": 0,
        "current_height": 295384,
        "current_state_hash": "bd04f38fbedcae7ce088955dc01d2cab5bea1f5af118d7134aa4047b3a726803",
        "difficulty": 776.5170288085938,
        "maturing": 0,
        "prev_state_hash": "f2f099fd72959750ee4c4a96b588b0a657f16562e7c3acded1d946ec8957b4ac",
        "receiving": 0,
        "sending": 0
    }
}
```

**Platform and build**
mac 3.1.5737

**Comment by nesbox** [comment]
fixed with #865 here b857502

**Comment by andreibaranouski** [comment]
verified with the latest master
